x-pack/plugins/stack_connectors/public/connector_types/jira/jira_params.tsx

```diff
@@ -381,6 +383,51 @@ const JiraParamsFields: React.FunctionComponent<ActionParamsProps<JiraActionPara
             }
           )}
         />
+        <EuiFormRow
+          fullWidth
+          error={errors['subActionParams.incident.otherFields']}
+          isInvalid={
+            errors['subActionParams.incident.otherFields'] !== undefined &&
+            errors['subActionParams.incident.otherFields'].length > 0
+          }
+        >
+          <JsonEditorWithMessageVariables
+            messageVariables={messageVariables}
+            paramsProperty={'otherFields'}
+            inputTargetValue={actionParams.subActionParams?.incident.otherFields || undefined}
+            errors={errors.otherFields as string[]}
+            label={
+              <>
+                {i18n.translate('xpack.stackConnectors.components.jira.otherFieldsFieldLabel', {
+                  defaultMessage: 'Additional fields',
+                })}
+                <EuiIconTip
+                  size="s"
+                  color="subdued"
+                  type="questionInCircle"
+                  className="eui-alignTop"
+                  data-test-subj="otherFieldsHelpTooltip"
+                  aria-label={i18n.translate(
+                    'xpack.stackConnectors.components.jira.otherFieldsHelpTooltip',
+                    {
+                      defaultMessage: 'Additional fields help',
+                    }
+                  )}
+                  content={i18n.translate(
+                    'xpack.stackConnectors.components.jira.otherFieldsHelpText',
+                    {
+                      defaultMessage:
+                        'When adding custom fields, please note that the system does not validate inputs or enforce required fields. To avoid failed actions, ensure compliance with Jira policies.',
```

Comment:
@lcawl @mdefazio Could you please review this tooltip text/design? It's like a helptext but as the content is too long, we'll include it as a tooltip. It will look like this:
<details>
  <summary>Screenshot</summary>
<img width="940" alt="Screenshot 46" src="https://github.com/elastic/kibana/assets/17549662/3819c62a-002e-456f-8635-eeb757e6eb28">
</details>